packages/core/src/client.ts

```diff
@@ -2267,11 +2268,15 @@ export class MedplumClient extends EventTarget {
     const retryDelay = 200;
     let response: Response | undefined = undefined;
     for (let retry = 0; retry < maxRetries; retry++) {
-      response = (await this.fetch(url, options)) as Response;
-      if (response.status < 500) {
-        return response;
+      try {
+        response = (await this.fetch(url, options)) as Response;
+        if (response.status < 500) {
+          return response;
+        }
+        await new Promise((resolve) => setTimeout(resolve, retryDelay));
+      } catch (err: any) {
+        this.retryCatch(retry, maxRetries, err);
       }
-      await new Promise((resolve) => setTimeout(resolve, retryDelay));
```

Comment:
This line effectively sleeps for `retryDelay` milliseconds.  I think It belongs here, outside of the `try/catch` rather than inside.

As currently written, the code will not sleep on error.

Reply:
@codyebberson We can discuss this offline if needed. I was trying to add the `retryDelay` and it broke a lot of tests. After doing some more investigation, I don't think the this line ever gets called, even before I added the `try/catch`, so I effectively removed it completely. Let me know your thoughts